Sources/Router/Presenter/Presenter.swift

```diff
@@ -3,11 +3,17 @@ import SwiftUI
 @available(iOS 13, macOS 10.15, *)
 public protocol Presenter {
     associatedtype Body: View
-    var replacesParent: Bool { get }
+    var presentationMode: RoutePresentationMode { get }
     
     func body(with context: PresentationContext) -> Body
 }
 
+public enum RoutePresentationMode {
```

Comment:
Please make this an `internal enum` wrapped by a `struct` with static properties. That allows us to add more presentation modes without breaking API.

Reply:
Please see my comment on why I think this is a better solution.

Reply:
After discussion, this has been moved to #4

Reply:
Added it more explicitly to #4 after discussion